### ISSUE TITLE
Oi, did you slow down programming?

### ISSUE THREAD

**Issue by IAFahim** [BushraOnika/CSE434#1]
I had a dream that you were getting married and was like I should check up on her (not because I was jealous or anything at alll). You you tell tell why are you stopping humm). Like you shouldn't you be excellent at your field so that you have lots of option in your life naa.

**Comment by IAFahim** [comment]
I heard from a friend of mine that you were trying to join a university abroad. So you might not visit this account naa, so i wanted to say sorry. Might not mean anything to others but wanted to say it huh. Be well and never do anything that harm yourself, and always take care of yourself, learn how to swim (its really really easy) 60% of you is water and due to lunghs full of air is possible you won't drown and be afloat with minor hand and leg movement, hope everyone at your home always stay healthy. Allah Hafez.

**Comment by BushraOnika** [comment]
When I lost one of my excellent friend that time I lost my will power for
coding.
Btw I'm not getting married but yes hopefully next year I will leave
Bangladesh if Allah wants.
And who is that friend cause I didn't told this anyone except some specific
close friends??? 🌚
Btw congratulations you got your dream job!!

**Comment by IAFahim** [comment]
You are aliveee

**Comment by IAFahim** [comment]
He wasn't that good of a friend huh

**Comment by IAFahim** [comment]
Thank you so much for replying

**Comment by IAFahim** [comment]
His other friend told him, I forget the name

**Comment by IAFahim** [comment]
You are really really quick learner so you should code naa, your friend would have wanted you to be super successful in life

**Comment by IAFahim** [comment]
But as you are leaving for higher studies teacher there would give you lots of boka and you would have tooo code. Nice teachers

**Comment by IAFahim** [comment]
Is your ammu okay? What happened afterwards?

**Comment by BushraOnika** [comment]
> Is your ammu okay? What happened afterwards?

Her operation done in 2022 and now she is fine absolutely.

**Comment by IAFahim** [comment]
Did you know it's really really painful when thinking about someone and there are chance that they person might never reply back. Or get a chance to see the text. I was thinking whole day were you might have be and what you might be doing. I really hate human feelings hurts a lotttt. Ehhh

**Comment by IAFahim** [comment]
> > Is your ammu okay? What happened afterwards?
> 
> Her operation done in 2022 and now she is fine absolutely.

Allhamdulliah

**Comment by IAFahim** [comment]
You at home? Is your capstone project near?

**Comment by BushraOnika** [comment]
You shouldn't thinking about me as you said before remembered? 
Why are asking your friends or whoever if I'm leaving or not?

**Comment by IAFahim** [comment]
And who told you about my job? Wait did you did tutoring like you wanted to

**Comment by BushraOnika** [comment]
> You at home? Is your capstone project near?

My capstone is already done by last semester. I have graduated from nsu

**Comment by BushraOnika** [comment]
> And who told you about my job? Wait did you did tutoring like you wanted to

I always keeps my friends updates whatever they accept my friendship or not

**Comment by IAFahim** [comment]
> You shouldn't thinking about me as you said before remembered? Why are asking your friends or whoever if I'm leaving or not?

It was really nice knowing something about you out of no where. I worry huh

**Comment by IAFahim** [comment]
> > You at home? Is your capstone project near?
> 
> My capstone is already done by last semester. I have graduated from nsu

You finished university before me. Mashllallah

**Comment by IAFahim** [comment]
> > And who told you about my job? Wait did you did tutoring  you wanted to
> 
> I always keeps my friends updates whatever they accept my friendship or not

I didn't accept it because if I don't keep it your life would be very different, huh

**Comment by BushraOnika** [comment]
> > > You at home? Is your capstone project near?
> > 
> > 
> > My capstone is already done by last semester. I have graduated from nsu
> 
> You finished university before me. Mashllallah

And you got your 1st job and that is also your dream job MashAllah. May Allah bless you like this always

**Comment by IAFahim** [comment]
> And you got your 1st job and that is also your dream job MashAllah. May Allah bless you like this always

And I never told you about my dream job, how do you know so muchh

**Comment by IAFahim** [comment]
You have a cattttt

**Comment by BushraOnika** [comment]
> > And you got your 1st job and that is also your dream job MashAllah. May Allah bless you like this always
> 
> And I never told you about my dream job, how do you know so muchh

I'm a detective as you know 🕵😂

**Comment by IAFahim** [comment]
Ow ow I once opened Whatsapp and saw you in shari, mashallah you looked really nice. Then I was like yeapp not going to use WhatsApp again

**Comment by IAFahim** [comment]
> I'm a detective as you know 🕵😂

And I was worried that you might not be taking care of your nice brain

**Comment by IAFahim** [comment]
What you going to study abroad? Which country?

**Comment by BushraOnika** [comment]
> You have a cattttt

Yes he lives in stray and we feeding him hen one day come to our house with broken leg and we adopt him

**Comment by IAFahim** [comment]
Does parrot fight with him?

**Comment by BushraOnika** [comment]
> Does parrot fight with him?

Nope that parrot died

**Comment by IAFahim** [comment]
How?

**Comment by BushraOnika** [comment]
> How?

One cat bite him

**Comment by IAFahim** [comment]
Was your brother angry when you adopted the cat then?

**Comment by BushraOnika** [comment]
> Was your brother angry when you adopted the cat then?

Nope that cat is not the one we adopted

**Comment by IAFahim** [comment]
I thought he might start hating cat afterward

**Comment by IAFahim** [comment]
Do you know i told a senior apu at our office about you and that i didnt even say sorry for hurting you.

**Comment by IAFahim** [comment]
You were alawys was so supportive of me, and I am really selfish. Can I ask you to forgive me? Ahem and start coding

**Comment by IAFahim** [comment]
And again, thank you for replying to my text. I am not trying to disturb you. And if I am disturbing you good huh. I didn't reply to any of your text and handwritten note because I wanted you to hate me.

**Comment by BushraOnika** [comment]
Nope I forgive you that time only no need to ask for it.Don't worry about it you didn't hurt me it's ok. Live your  as you wish.. 
Will try to focus

**Comment by IAFahim** [comment]
Wait i didn't hurt you ehh. And I tried so best

**Comment by IAFahim** [comment]
Hayyy `Live your as you wish` i had super nice master plan, huh.
What was your capstone project?

**Comment by IAFahim** [comment]
Thank you for being nice till the end. Does it feel bad always being the nice person in the conversation? I would be going away now. I really wanted to say sorry. I am sorry didn't explain why I have been such a bad person. In after, life will tell you. Take care, may Allah bless you.

**Comment by BushraOnika** [comment]
Whatever I don't remember things that hurts me and I have no hard feelings for you as well. I believe may be I was faulty for all those things. And you have your own reasons so no hard feelings or no nice person also that's how I'm I can't do anything about that.

**Comment by BushraOnika** [comment]
> Hayyy `Live your as you wish` i had super nice master plan, huh. What was your capstone project?

I still don't know what was the project btw cause I didn't do anything in that my teammates were too much good that's it they do that whatever that was

**Comment by BushraOnika** [comment]
Live your life*🌚 as you wish ehhhh

**Comment by IAFahim** [comment]
> Live your life*🌚 as you wish ehhhh

Won't huh

**Comment by IAFahim** [comment]
> > Hayyy `Live your as you wish` i had super nice master plan, huh. What was your capstone project?
> 
> I still don't know what was the project btw cause I didn't do anything in that my teammates were too much good that's it they do that whatever that was

So Bad bad

**Comment by BushraOnika** [comment]
Not interested in coding and that project also I didn't like the idea of our project :"")

**Comment by IAFahim** [comment]
> Whatever I don't remember things that hurts me and I have no hard feelings for you as well. I believe may be I was faulty for all those things. And you have your own reasons so no hard feelings or no nice person also that's how I'm I can't do anything about that.

You weren't faulty at all

**Comment by IAFahim** [comment]
> Not interested in coding and that project also I didn't like the idea of our project :"")

Was it something AI related?

**Comment by BushraOnika** [comment]
Which faculty under you are doing your capstone??

**Comment by BushraOnika** [comment]
> > Not interested in coding and that project also I didn't like the idea of our project :"")
> 
> Was it something AI related?

Skin cancer deep learning something 🙂

**Comment by IAFahim** [comment]
> Which faculty under you are doing your capstone??

Mohammad Ashrafuzzaman Khan

**Comment by IAFahim** [comment]
> > > Not interested in coding and that project also I didn't like the idea of our project :"")
> > 
> > 
> > Was it something AI related?
> 
> Skin cancer deep learning something 🙂

I guessed that much, youtube tutorial stuff

**Comment by IAFahim** [comment]
You are not faulty, i heard you say it a few times before. Why do you think that?

**Comment by IAFahim** [comment]
Like everyone loves you. You helped so many people at university. If you had some fault, others would have told you by now, right?

Why do you think it was your fault?

**Comment by BushraOnika** [comment]
> Like everyone loves you. You helped so many people at university. If you had some fault, others would have told you by now, right?
> 
> Why do you think it was your fault?

Forgot those things they don't matter anything now at all. Wish you a great life ahead with you dreams

**Comment by IAFahim** [comment]
Safe journey

**Comment by BushraOnika** [comment]
As you are now established and have a job do the marriage fast fast and forgot the mistakes you had done in past and learn from them and don't repeat them in future. 
And Keep me in your prayers so that I can also achieve something in life😅

**Comment by BushraOnika** [comment]
> Safe journey

And one advice just don't run from your life. Give life a chance may Allah have written a beautiful story for you if you run before entering how you would ever know!!!! 
And just don't break any relationship because of your assumptions cause you don't know how the other actually feels and want!!

**Comment by IAFahim** [comment]
I am not getting married. But I would pray you become very successful in your field

**Comment by BushraOnika** [comment]
> What you going to study abroad? Which country?

Hopefully UK or some European country don't know after giving IELTS can decide better

**Comment by IAFahim** [comment]
> > Safe journey
> 
> And one advice just don't run from your life. Give life a chance may Allah have written a beautiful story for you if you run before entering how you would ever know!!!! And just don't break any relationship because of your assumptions cause you don't know how the other actually feels and want!!

Boka you, you had marriage proposal from US and you are really good-looking, and slow caring being with me would have only got you like what my 40K salary. It's not enough to run a family. I heard love fades after 5 years or after that, its being dependent on each other. There aren't much that i can offer nor my family that rich. So techianlly you would be better of with a suitable partner that your dad would pick. Plus i don't want my love to fade away. So my plan was to create a copy of you in brain and talk with her distrube her alot. But problem is, brain you cant reply at. I can just tell jokes and say something to you. That way i don't disturbe your time and you become happy in life. If i kept you, wouldnt that be more selfish? like with you i would have got the most pretties nices persion in the world, and you would have gotten me only. And to this date i can't get out of this logic so i never disturbed you.

**Comment by IAFahim** [comment]
> > What you going to study abroad? Which country?
> 
> Hopefully UK or some European country don't know after giving IELTS can decide better

You are really fluent in english currenly IELTS would be easy. UK Visa takes around 4 month if you get a job before trying to go for higher stuides. One person who works in out office appied but scince there is a problem of over migration problem in UK, its been 4 month and still he didnt get visa. And his class started a month ago and he lost 2 university admmission.

So Apply for fast visa and May allah not make you go though what he is facing. Some good news is other student who applied before him got visa. So it might be becasue of his job, as a lot of student goes UK and drop out and just get a job.

**Comment by IAFahim** [comment]
AI can code better than all my Colleague at office. I am just like 20%-30% better than AI, and it's really closing in really fast. So I have to code every day so that I don't become irrelevant in this industry.

**Comment by IAFahim** [comment]
> And one advice just don't run from your life. Give life a chance may Allah have written a beautiful story for you if you run before entering how you would ever know!!!! And just don't break any relationship because of your assumptions cause you don't know how the other actually feels and want!!

Let's say if you know that there might be someone better for the person, like there are possibly lots of people who can be a better fit for a person right? We meet because of you and you were so nice to me that I thought I found the perfect person in the world right. But what if I am not a perfect person, so I thought then I would be the perfect person for that person. So I kept trying, trying to understand how you think what makes you act so kindly to others, and I started understanding some parts of it, I used to try doing the same with others, my ammu was really impressed with the new me. But I never understood why you talked with me in return. Every day when talking with you I found out how caring you are but I am pretty sure that I am really rude person. So I thought do I use you to try to make myself better? Like for everything I learned what do I provide why should I get to be with you. Then I knew every marriage has lots of fighting, there is a period of time that people like each other and after that everything they loved about the other person starts to be normal. And it fades over time, so am I just being with you to make myself happy, like love works with proximity right, that's why we don't start like chatting with someone other side of the world. Like I tried to understand what do want from other person in return, and there was nothing, I might have never found someone who is so selfless as you. You don't want anything other then the other person to be near you. Like so easy naa. Then I thought wait anyone in the world can provide that, like anyone. But what about when loves fades wouldn't it come down providing for each other. And yes, I was little better than others in programming so I thought it's possible that I would try to be that person, so that I can always be with you. But then I thought wait, I love you for lots of reasons, like you are tall, good looking, always taking with me like I am the most important person in the world, not crazy, have GitHub account, super duper smart loved by everyone, and like onek onek cute. But anyone in the world would be happy to find someone like that. And isn't that make my reason of liking you conditional. Like if you were someones wife back then I wouldn't have started liking you. So like I am quite selfish right? Loving someone because of conditions of I want her in my life or I wouldn't love them. But shouldn't I be someone who help her being successful in life, giving the best life possible. And also do I only love her because I want to be with her! So I thought what if I just be her friend and help her being super smart and try to find her the best person in the world, then I could never be selfish right! So I told you we would just be friends. And afterwards I was like it hurts so much saying I would just be her friend, it was really really painful. So close to someone knowing you would lose her some day was really painful naa. But I thought if I feet this strong now how would I feet if I thought of it 3 years after. So I kept going and always tried to keep my distance from you, and started feeling you getting sad also, I thought if I went away totally and if you found someone else or someone else finding you as they would for sure if they talked with you once and your abbu is there to give boka if the other person is bad at all so that wouldn't be a problem and you would definitely find a nice person then. Would then you won't be sad anymore. Then one day you were like super sad, and I was like it's all my fault, see now she is sad sad. And I was like if she is sad because of me and if I go away she won't be sad anymore right? Then I remember the 5 stages of grief.

Like there might be 5 stages of grief:
denial, anger, bargaining, depression, accpectence. 

If I just make you angry and don't explain anything there won't be a stage of denial, anger or bargaining right. But there would be a problem of depression, and I thought if she find someone quick enough she might not be depressed, and technically other would find her. So she would be in accpectence right! So I didn't say anything and started not replying to you so that you feel really angry at me. I tried my best to hurt you as much possible so that you never even talk with me. And I knew the little help I did to tech little things I thought you would kept learning focusing on yourself and be super duper smarty. But it wasn't the plan to make you sad and not to focus on your carrier anyone. Are you crazy all that hard work and me being sad sad and having to talk with a copy of you for like 2h on average every day mainly saying sorry in my brain and you stopped practicing. So I came to give you lots of boka. How rude. All my secret plan and you not focusing on self. I hope you found someone new, someone how doesn't live because of only lust. Ow ow jano, I asked my friends if they would marry someone but they weren't allowed to touch her, and everyone was like noo way. And I am like see that's not like love at all. Shouldn't love be about caring for other persons happiness and helping the lift up. 

See I have a video https://www.youtube.com/watch?v=yJSiUm6jvI0

Like see that's like love right. Youuu I am warning you who ever you find you two better try to help each other being a better. Or else I would have to [insert killing gif, or scary knife here...]. See i am not trying to disturb you. Tell the person who like currently to help you being better. Huh rude rude. Tata

I am greatful to Allah that I get to talk with you once. 
And I am sorry for not considering your emotions, I should have asked you what you felt.

**Comment by IAFahim** [comment]
I am sorry for making you feel bad. Really really sorry. I always bottled up the pain as i had the upper hand, but I never knew it hurts this bad. I am sorry for making you go through it

**Comment by BushraOnika** [comment]
No its fine why are you saying sorry I didn't feel bad at all. Everyone has their right to choose their life and how will they live it. It's totally fine. 
You are studying and also doing 40k job its alot you know MashAllah👀
And I'm graduated with no jobs and if I got a free internship also I would do that happily🙂
Btw nice idea come I need a internship I'm feeling bored by sitting home🙂🙂

**Comment by BushraOnika** [comment]
Now Give me some names where I can do an internship without any skill😂😂

**Comment by IAFahim** [comment]
> No its fine why are you saying sorry I didn't feel bad at all. Everyone has their right to choose their life and how will they live it. 

Did you know you were exactly this forgiving when I talked with you. Like, I have no idea how is a person so caring about others feeling. I really don't know how you forgive people so easily. May Allah always reward you for your kindness. And I am still sorry, huh. Never knew moving on gives this much pain. I heard about how painful it is, that's why on the first time we talked and found out how nice you were, I was like I would totally start liking you, so not going to talk with you at all. But good you did, you thought me so much in that I am going to carry over my entire life.

> You are studying and also doing 40k job its alot you know MashAllah👀

With 40k, 10K goes away because of travel and lunch, breakfast, and 12K for family maintenance. Then left 18k. Living cost in Bangladesh is very High. But yes MashAllah. I know most people have trouble getting a job. And I didn't have to give an interview and got selected due to covid, everyone got online and every company needing programmer.

>And I'm graduated with no jobs and if I got a free internship also I would do that happily🙂
Btw nice idea come I need an internship I'm feeling bored by sitting home🙂🙂

Due to current AI hype, no one hires anyone without skills or years of experience.
You have 3 option.
1. You can do Competitive programming, as it going to help you a lot in university (please trust me).
2. Learning something from an expert.
3. Tutoring (other than meow)

You aren't still confident in programming yet, but you are super smart, so you can learn in no time. I have a question. You know, girls don't do good at programming because they feel it lacks human interaction. So if you tell me how you are spending days and what part of computer science makes you happy or interesting. I think I can trick your brain into liking that field.

I would suggest you not to get internships till joining UK university, as currently if they notice that you are applying for jobs and internships and not focusing on study the visa gets delayed as for my experience with a colleague at office, who is stuck where all his other friends got visa. 

And I just checked your LinkedIn and you are wearing shari are you crazy. And know you are already learning we development, allhamdulliah. I could can give you a job, but I don't think you would be able to perform well from start. And you have grown to be more beautiful I think. Sorry, off topic. You should really focus on yourself and learning because internships are quite time consuming and the pay is really low and the traveling it cost most of the earnings around 50%. Tutoring would be the best (but I am biased as I have experience in it). But pick your option, as liking what to do have positive impact in life.

**Comment by BushraOnika** [comment]
So rude you didn't give me job😞😞
 Me now being sooo nice to you but now also you didn't give you didn't changed always rude rude :"")

**Comment by BushraOnika** [comment]
Whatever in computer science I just like machine learning cause that's the easy one project I had done 🙃🙃

**Comment by IAFahim** [comment]
> So rude you didn't give me job😞😞 Me now being sooo nice to you but now also you didn't give you didn't changed always rude rude :"")

I tried to tell you the truth naa. Job is going to be a huge part of your life. And if I lie or don't point out how bad it might sound nice nice. But what if you find out the hard way.

**Comment by IAFahim** [comment]
Do you want part of my job? I can give it to you for 100 BDT per commit? Like every time you upload a code you can buy two ice cream.

**Comment by BushraOnika** [comment]
Slow down slow down I was just joking but as always you are taking seriously ohooooo🤦‍♀️

**Comment by IAFahim** [comment]
> Whatever in computer science I just like machine learning cause that's the easy one project I had done 🙃🙃

First tell me what you like about computer science? And where do you want to see yourself doing in future.

**Comment by BushraOnika** [comment]
> Do you want part of my job? I can give it to you for 100 BDT per commit? Like every time you upload a code you can buy two ice cream.

Ohoo nice trade offer will be look onto it 👀👀

**Comment by IAFahim** [comment]
> Slow down slow down I was just joking but as always you are taking seriously ohooooo🤦‍♀️

I apologize. I don't know how to fix it 🥲

**Comment by BushraOnika** [comment]
> > Whatever in computer science I just like machine learning cause that's the easy one project I had done 🙃🙃
> 
> First tell me what you like about computer science? And where do you want to see yourself doing in future.

Nothing... 
In future I see myself in different different countries for tour👀

**Comment by IAFahim** [comment]
> > Do you want part of my job? I can give it to you for 100 BDT per commit? Like every time you upload a code you can buy two ice cream.
> 
> Ohoo nice trade offer will be look onto it 👀👀

You would have to be really smarty really quickly can you handle that?

**Comment by BushraOnika** [comment]
> > > Do you want part of my job? I can give it to you for 100 BDT per commit? Like every time you upload a code you can buy two ice cream.
> > 
> > 
> > Ohoo nice trade offer will be look onto it 👀👀
> 
> You would have to be really smarty really quickly can you handle that?

Nope I'm old now my brain works slow🐌 😞
But but you should consider me as you said I'm good at heart 👀

**Comment by IAFahim** [comment]
> Nothing... In future I see myself in different different countries for tour👀

ML and web development both is either remote or office based work. You would have to join multinational company that has multiple Branch in different country and you are so smart at consultation that they move you around.

You then need remote job, so you can go on tour but would have to take work with you there. Ehh, there are job that make you travel but I think Allah need to put it in your luck to place you there. So hopefully we can pary and you can practice a lot so that you become super good.

**Comment by IAFahim** [comment]
How much do you want to earn?

**Comment by BushraOnika** [comment]
> How much do you want to earn?

Whatever I will got I will be happy 😌
I don't need much things or money to happy huh

**Comment by IAFahim** [comment]
> Nope I'm old now my brain works slow🐌 😞
But but you should consider me as you said I'm good at heart 👀

Then the rule is as follows for the job
The first month you would have to understand how Unity and game development works. For that you would have to learn C# first. For first month I would give you 5000 BDT. As you won't be able to contribute much. 

After that I would ask you to develop part of a game with fixed amount of library and using a specific flow.

For each commit I would pay 100 BDT. That would make you around 15k per month. 

And also I would give you 20 taka for every consicutive day you push code, as you are so nice in heart 😌 which you are.

But as I am really really bad bad, _muh ha ha ha_ I would cut 200 taka per day for time it takes to finish a task.

Do you accept? You can give counter offer !!

**Comment by BushraOnika** [comment]
Wait are you opening a company or something??? 
How would you pay me?? :"") 
Its totally fine you don't need to pay me so much for doing nothing 🤦‍♀️🤦‍♀️
You don't know business aha you face loses if you offer this way everyone 🤦‍♀️

**Comment by IAFahim** [comment]
> Whatever I will got I will be happy 😌 I don't need much things or money to happy huh

Allhamdulliah.

On interview don't say that, the first thing I learned the hard way. I was evaluated at 160k per month and instead of asking around that. Was like sure give what ever you like. And they thought I didn't know work maybe so I can't tell them a clear number or how much I am worth. So I got 30k at start and 15% salary increase per year.

**Comment by BushraOnika** [comment]
> > Whatever I will got I will be happy 😌 I don't need much things or money to happy huh
> 
> Allhamdulliah.
> 
> On interview don't say that, the first thing I learned the hard way. I was evaluated at 160k per month and instead of asking around that. Was like sure give what ever you like. And they thought I didn't know work maybe so I can't tell them a clear number or how much I am worth. So I got 30k at start and 15% salary increase per year.

Ohhhhh 😱😱😱😱 
So sweet of youuuu ok I will remember this one next time

**Comment by IAFahim** [comment]
> Wait are you opening a company or something??? How would you pay me?? :"") Its totally fine you don't need to pay me so much for doing nothing 🤦‍♀️🤦‍♀️ You don't know business aha you face loses if you offer this way everyone 🤦‍♀️

Its a business but I have lots of task at hand and no one at my office can come close to what I can do, so every hard task is giving to me, and time passed doing that and no one did any hard people and I always fixed them making me more smarty smarty and make others slack off as hard thing can be done by me so they didn't even learn a single hard thing, so they come to office and watch tv series all day and whole company is like me and everyone managing me. I tried to teach them but why would they understand hard stuff when they can watch movies. So last month I said I want to quit. And the everyone at whole office started panicking. As no one can understand what I wrote nor they can do my task. So after calling my mom that I shouldn't quit by the CEO, ammu was like do the work, as they are asking so much. But I also have capstone so I become fully offline now I work from home. So if I can find someone who can take part of my burden I am planning to give my full job to him. And if you are super smart I can directly give my job to you as I want to be free of work for once. So it's not bad at all huh. And you would also have to become super smart so it would work out really really well.

**Comment by BushraOnika** [comment]
> > Wait are you opening a company or something??? How would you pay me?? :"") Its totally fine you don't need to pay me so much for doing nothing 🤦‍♀️🤦‍♀️ You don't know business aha you face loses if you offer this way everyone 🤦‍♀️
> 
> Its a business but I have lots of task at hand and no one at my office can come close to what I can do, so every hard task is giving to me, and time passed doing that and no one did any hard people and I always fixed them making me more smarty smarty and make others slack off as hard thing can be done by me so they didn't even learn a single hard thing, so they come to office and watch tv series all day and whole company is like me and everyone managing me. I tried to teach them but why would they understand hard stuff when they can watch movies. So last month I said I want to quit. And the everyone at whole office started panicking. As no one can understand what I wrote nor they can do my task. So after calling my mom that I shouldn't quit by the CEO, ammu was like do the work, as they are asking so much. But I also have capstone so I become fully offline now I work from home. So if I can find someone who can take part of my burden I am planning to give my full job to him. And if you are super smart I can directly give my job to you as I want to be free of work for once. So it's not bad at all huh. And you would also have to become super smart so it would work out really really well.

You should apply somewhere else where you can have some free time and a much good salary as you said you are already doing everyone's job so you should be paid more

**Comment by BushraOnika** [comment]
And about me I'm not that much intelligent as you are that I can take your job 🤦‍♀️🤦‍♀️
I have literally 0 knowledges now in coding

**Comment by BushraOnika** [comment]
And if you give this offer of earning by only learning I believe everyone will like this one and want to grab this opportunity so you should give this offer someone intelligent and smart as you are 
Not someone like me who have no knowledge!! 🤦‍♀️

**Comment by IAFahim** [comment]
> And about me I'm not that much intelligent as you are that I can take your job 🤦‍♀️🤦‍♀️ I have literally 0 knowledges now in coding

Good good then you can only get better not worse, pew I thought what is you were like 100 and was 60 at job

**Comment by IAFahim** [comment]
> And if you give this offer of earning by only learning I believe everyone will like this one and want to grab this opportunity so you should give this offer someone intelligent and smart as you are Not someone like me who have no knowledge!! 🤦‍♀️

I did to 3 person already they all came took money did zero task still getting paid 15k each. (Different easier side project that's going on)

Youuu try try naa. Don't you remember how good you were, like the first person who I might have got jealous of getting better grade them me. Lucky it allowed in Islam to get jealous in study. 

Back then I was sure I can never be better student then you. Good you know you are going aboard to study means you would always be better then me and I can be always jealous huh. Need a friend to be jealous of huh huh. 

Do you get afraid of what the code is doing under the hood. Like I could never find out, why you get scared little little when you programmed. I thought you would get over it if you kept practicing for a while. I should have asked you what you think when you code and how you approach it. But I learned with trying and failing at first so I thought if I gave you enough time to fail and advice I give would be more impactful. Which I think wasn't the the best approach for you, I am sorry for that. See not that wise at all. So you can definitely do my job. Wanna try? Just google C# and practice online. It's like Java, but better, and fun.

**Comment by BushraOnika** [comment]
Do you pay them with your salary?? 👀
And why you pay them when they don't work?? 😒
Ok I will try c#  hopefully it will works.

**Comment by IAFahim** [comment]
Ow ow and thank you for replying. I am like blabbing a lot. You you don't have to listen to me you know and also don't have to reply. I was like so grateful for one text from you, and you worte so many. I can't never repay you in first place so I don't know what would do with so many blessing, I would have never be able to repay you for one thing and you always keep being nice nice. You should not reply huh. You should live your life and be happy naa. Always helping others. I am so sorry for every time I made you sad, I left so that I never want to hurt someone I like so left everything fades away right. You should be happy in life only.

**Comment by IAFahim** [comment]
> Do you pay them with your salary?? 👀 And why you pay them when they don't work?? 😒 Ok I will try c# hopefully it will works.

When you hire someone it's really difficult to fire someone, because as boys there life fully revolves around the money, like they can't even pay rent or have basic things in life because father treats differently to middle class people. 

But good thing is the company pays. I just don't take the money for the work. It's companies responsibility to handle them. I just find them and hand them over so the company doesn't disturb me.

**Comment by IAFahim** [comment]
Okay to ahead and start I would send you the first payment 13th November. Do well, or elseeee muh ha ha. Or else nothing I am just trying to scare you. But or elsseeee muh ha ha. Or am I?

**Comment by IAFahim** [comment]
Since you have accepted it, I can tell you now the other 3 works on side project. I work on the main one. So technically you would get salary from my part. Do a good job and my full salary is yours and I can finally after 2 year rest and work on my portfolio and my ideas. And also good thing if you needed more encouragement that you don't technically don't own any job so you can apply easily for UK. With the added benefits of fully doing job and getting experience, where if you even fail, I can take over and finish it. But you you remember it might create bad habits so you should try your bestest huh. And also focus on IELTS.

**Comment by IAFahim** [comment]
> Since you have accepted it, I can tell you now the other 3 works on side project. I work on the main one. So technically you would get salary from my part. Do a good job and my full salary is yours and I can finally after 2 year rest and work on my portfolio and my ideas. And also good thing if you needed more encouragement that you don't technically don't own any job so you can apply easily for UK. With the added benefits of fully doing job and getting experience, where if you even fail, I can take over and finish it. But you you remember it might create bad habits so you should try your bestest huh. And also focus on IELTS.

**Comment by BushraOnika** [comment]
Hey I don't need the salary I just need some guidelines that's it enough for me so don't take burden that you have to pay me . 
And I just want to gather some experiences so that when I go abroad I can earn my living cost and my meal costs also that's it. Cause I have to do part time jobs there for my expenses. 
No need to pay its fine

**Comment by BushraOnika** [comment]
And and tell me what are the things I have to learn in C# :"))

**Comment by IAFahim** [comment]
When you go aboard, in UK you aren't allowed to work more than 20h a week. So you would have to do get paid doing drom duty, part time job, or best would be internship. And yes for internship it might be helpful to know some programming. But you why not learn by doing work that would be part of a end product hmm, and it's not like I am giving it for free 😌 it would be hard huh and what if you run away! so giving you 5k to learn so that you don't run away. Our office gave us 3 month for free without any responsibility, and I am only offering 1 month, plus my work load would reduce huh. 

You have big heart so you say no right naa, but I am very bad so I should say, yes you work for free. So I am saying let's switch place for a little bit and you take money and work onekk hard and I rest. Not a bad deal at all. 🙂 So you are taking it. Every company does it. 

If you want to learn there are lots of C# videos, watch and practice. I would start from basic. After you understand a little bit I would tell you how it works. And everything would be easy. I hope. If I give you learning link you have to watch them, would you feel like it's like a me putting you upto something by force?

**Comment by IAFahim** [comment]
> Hey I don't need the salary. 

Remember I told not to say that you even thanked for it, already forgot. And also when a senior person says some you should do. Removed it, Dictator mode starts.
![](https://media3.giphy.com/media/DqNeh0L4NdPX28pWdO/giphy.gif?cid=6c09b9529hf8quw19sv8vkxsgtl60sy4z39ohbzk4en0ldt2&ep=v1_internal_gif_by_id&rid=giphy.gif&ct=g)

You have bank account? Ask your parents account number I would forward it there.

**Comment by BushraOnika** [comment]
If I ask them accounts number they would ask so many questions what would I reply then?? 🙂


> > Hey I don't need the salary.
> 
> Remember I told not to say that you even thanked for it, already forgot. And also when a senior person says some you should do. Removed it, Dictator mode starts. ![](

Ok boss done as you are senior citizen I listen your all commands🙂🙂

> You have bank account? Ask your parents account number I would forward it there.

They would ask alot of questions what would I tell them then??? Suggest that also that's why saying no need of money 🙂

**Comment by BushraOnika** [comment]
> When you go aboard, in UK you aren't allowed to work more than 20h a week. So you would have to do get paid doing drom duty, part time job, or best would be internship. And yes for internship it might be helpful to know some programming. 
> 
>
Yes that's the reason I was looking for internship type jobs I had asked you 1st want to gain some experiences for my living abroad as you know it very well...

**Comment by IAFahim** [comment]
> They would ask alot of questions what would I tell them then??? Suggest that also that's why saying no need of money 🙂

You should tell the truth, like it's a person who has job responsibilities and capstone so due to pressure I am doing the job part and getting paid for that part. 

> Ok boss done as you are senior citizen I listen your all commands🙂🙂

Ow yes yes, I shall be known as Captain. The journey starts.

**Comment by IAFahim** [comment]
> > When you go aboard, in UK you aren't allowed to work more than 20h a week. So you would have to do get paid doing drom duty, part time job, or best would be internship. And yes for internship it might be helpful to know some programming.
> 
> Yes that's the reason I was looking for internship type jobs I had asked you 1st want to gain some experiences for my living abroad as you know it very well...

Do your abbu having money problem?

**Comment by IAFahim** [comment]
Bank account is the safest rather than phone banking like bkash. Like if you aren't here and let's say you give me phone number of your dad and I started chatting with him instead and being best friends it would be more problematic ehh. You won't be here to defend it also.

**Comment by IAFahim** [comment]
Ow and some warning before we start. In world no one does anything without reason (without you maybe). So before I had a reason for being nice, but I always asked something in return.

But this time problem is I don't know why am doing what I am doing, so be very very cautious of me. As I might be planning something that even I don't know. 

Second I really like it when you text. Before I used to text you and go to other room and talk with others to pass time without  noticing for you to reply back. I used to always come to room either thinking about you or waiting for what you would say. I just did it am hour ago without noticing again. So try to not text me as it's quite alluring.

And what ever you do not be friendly with me, think of me as a dangerous entity. If it helps think of me as the person that trained a cat to kill your nice parrot. Yes that serious. 

And have a safe work environment. If I ever have to give link for chat, do not reply with voice. And try your best. And I can't 100% guarantee that what you would learn will for sure get you an internship. As I don't know about internships market well about that country but if I get information I would let you know.

**Comment by BushraOnika** [comment]
> > > When you go aboard, in UK you aren't allowed to work more than 20h a week. So you would have to do get paid doing drom duty, part time job, or best would be internship. And yes for internship it might be helpful to know some programming.
> > 
> > 
> > Yes that's the reason I was looking for internship type jobs I had asked you 1st want to gain some experiences for my living abroad as you know it very well...
> 
> Do your abbu having money problem?

Yeah whatever he had he will give for my higher study then there would be no savings or any money left so after that they had only one want that I took my brother cause all the savings and all the money is spending on me. So just they can afford the tuition fees and other things I have to manage by myself

**Comment by BushraOnika** [comment]
> Ow and some warning before we start. In world no one does anything without reason (without you maybe). So before I had a reason for being nice, but I always asked something in return.
> 
> But this time problem is I don't know why am doing what I am doing, so be very very cautious of me. As I might be planning something that even I don't know.
> 
> Second I really like it when you text. Before I used to text you and go to other room and talk with others to pass time without noticing for you to reply back. I used to always come to room either thinking about you or waiting for what you would say. I just did it am hour ago without noticing again. So try to not text me as it's quite alluring.
> 
> And what ever you do not be friendly with me, think of me as a dangerous entity. If it helps think of me as the person that trained a cat to kill your nice parrot. Yes that serious.
> 
> And have a safe work environment. If I ever have to give link for chat, do not reply with voice. And try your best. And I can't 100% guarantee that what you would learn will for sure get you an internship. As I don't know about internships market well about that country but if I get information I would let you know.

Ok done I understand no problem it would be professional only nothing more

**Comment by IAFahim** [comment]
That's a lot of responsibility. But if Allah keeps everything okay, your first 20 month salary would be enough for his whole tutoring and yours also

**Comment by IAFahim** [comment]
> > Ow and some warning before we start. In world no one does anything without reason (without you maybe). So before I had a reason for being nice, but I always asked something in return.
> > But this time problem is I don't know why am doing what I am doing, so be very very cautious of me. As I might be planning something that even I don't know.
> > Second I really like it when you text. Before I used to text you and go to other room and talk with others to pass time without noticing for you to reply back. I used to always come to room either thinking about you or waiting for what you would say. I just did it am hour ago without noticing again. So try to not text me as it's quite alluring.
> > And what ever you do not be friendly with me, think of me as a dangerous entity. If it helps think of me as the person that trained a cat to kill your nice parrot. Yes that serious.
> > And have a safe work environment. If I ever have to give link for chat, do not reply with voice. And try your best. And I can't 100% guarantee that what you would learn will for sure get you an internship. As I don't know about internships market well about that country but if I get information I would let you know.
> 
> Ok done I understand no problem it would be professional only nothing more

So rude, why nothing more huh.

**Comment by IAFahim** [comment]
As warned see alterior motive.

**Comment by IAFahim** [comment]
You can follow this, you get one week. Then i would tell you what I C# actually is and why it's one of the most fun languages to learn.
https://www.youtube.com/watch?v=YrtFtdTTfv0&t=1133s

**Comment by IAFahim** [comment]
Put what your learning in GitHub repo naa. So i can be like ow yes she is going up quite nicely.

**Comment by BushraOnika** [comment]
> Put what your learning in GitHub repo naa. So i can be like ow yes she is going up quite nicely.

Yeah will make a new repo and then will push there everyday learning from night.

**Comment by IAFahim** [comment]
Thank you for the consideration. When do you learn at Sadhinota Camp, and what do you learn there? And who do you spend your days?

**Comment by BushraOnika** [comment]
Nope I should thank you na for the opportunity??
O Sadhinota camp is basically is a course of full stack web development for 8 months which start at September just 1 month of the course I didn't learn much cause I don't think web development can be done by me at all😑

**Comment by IAFahim** [comment]
Then what do you do all day?

**Comment by IAFahim** [comment]
> Nope I should thank you na for the opportunity?? O Sadhinota camp is basically is a course of full stack web development for 8 months which start at September just 1 month of the course I didn't learn much cause I don't think web development can be done by me at all😑

You shouldn't thank, to tricking you huh. Into becoming better coder.

Do you like web development? Did you learn JavaScript a little?

**Comment by BushraOnika** [comment]
Sleeping, eating, playing with cat and taking IELTS preparation that's it

**Comment by IAFahim** [comment]
That's a lot of tasks. More questions: When do you wake you? Didn't you use to go our to relatives on Friday and Saturday?

**Comment by BushraOnika** [comment]
Hey you shouldn't ask me my personal life questions as you said we shouldn't be friends anymore...

**Comment by IAFahim** [comment]
I told you to be cautious, didn't tell you to follow the rules did I 😌 so answer quick quick

**Comment by BushraOnika** [comment]
I love sleeping so I wake up late

What are you making for your capstone project?

**Comment by IAFahim** [comment]
You would get sleep drunkenness. When do you go to sleep and when do you wake up? Tell me times. 

I am making an RPG game for capstone. 

Wait do you feel alone? Do you study for IELTS all day? What do you do for fun or when you are bored?

**Comment by BushraOnika** [comment]
No I don't study all day ewww just 2-3 hours maybe.....
And don't worry I'm not alone my some friends are also with me in this IELTS journey

**Comment by IAFahim** [comment]
Good, good I was thinking of it would be okay if I gave boka or not. Now I know
![Good to know](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExdWF4ZG9uNjV6bHVjYWc4N3YyMmJiZWZxemN0eHRyMzQycDA0dnQ2NCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/l4KhS0BOFBhU2SYIU/giphy.gif)
